templates/aide.conf.j2

```diff
@@ -0,0 +1,304 @@
+# Example configuration file for AIDE.
+
```

Comment:
If this file is not meant to be generated with users data, it can be put to `files` directory and copied to managed nodes with `copy` instead of `template`.

Reply:
still - nice to have it be a template so we can have the `{{ ansible_managed | comment }}` and `{{ "system_role:aide" | comment(prefix="", postfix="") }}` in the file for tracking.